modules/virt-about-services.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-creating-service-vm.adoc
+
+:_content-type: CONCEPT
+[id="virt-about-services_{context}"]
+= About Services
+
+A Kubernetes _Service_ is an abstract way to expose an application running on a set of pods as a network service. Services allow your applications to receive traffic. Services can be exposed in different ways by specifying a `spec.type` in the `Service` object:
```

Comment:
I wonder if "running on a set of pods" can be omitted here?

Reply:
Keeping this as is; this wording is preferred by the SME